assets/questions/findMostCommonRepeatedCharacter.js

```diff
@@ -0,0 +1,274 @@
+// Copyright 2018 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Most Common Repeated Character.
+ */
+
+globalData.questions['findMostCommonRepeatedCharacter'] = {  // eslint-disable-line dot-notation
+  title: 'Most Common Repeated Character',
+  starterCode: {
+    python:
+`def findMostCommonRepeatedCharacter(s):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`import collections
+
+class AuxiliaryCode(object):
+
+    @classmethod
+    def repeatsOnly(cls, s, ignore_case=True, reset_prev=True):
+      output = []
+      prev = None
+      for c in s:
+        if (prev and ignore_case and c.lower() == prev.lower()) or c == prev:
+          output.append(prev)
+          output.append(c)
+          if reset_prev:
+            prev = None
+        else:
+          prev = c
+      return ''.join(output)
+          
+    @classmethod
+    def overcountMoreThanTwoRepeats(cls, s):
+      repeats = cls.repeatsOnly(s, True, False)
+      if repeats:
+        return collections.Counter(repeats).most_common(1)[0][0]
+          
+    @classmethod
+    def dontIgnorePunctuation(cls, s):
+      repeats = cls.repeatsOnly(s)
+      repeats = [x for x in repeats if (not x.isalpha() and not x.isdigit())]
+      if repeats:
+        return collections.Counter(repeats).most_common(1)[0][0]
+    
+    @classmethod
+    def dontIgnoreCase(cls, s):
```

Comment:
I think that this is not much of a concern since we control the inputs. Which ever buggy output fails first will guide them towards solving one problem then the other.